src/app/rundown/components/segment/segment.component.html

```diff
@@ -2,7 +2,7 @@
   <div class="c-segment__header" (click)="setFirstValidPartAsNext()">
     <div class="c-segment__header-toolbar">
       <span>123</span>
-      <span style="margin-left: 50px;">00:30</span>
+      <span style="margin-left: 50px;" *ngIf="segment.budgetDuration !== undefined">{{segment.budgetDuration / 1000 | timestamp}}</span>
```

Comment:
Why 1000? Would be nice to extract this into a variable with a descriptive name :)